features/netsocket/nsapi_types.h

```diff
@@ -125,6 +125,8 @@ typedef enum nsapi_security {
     NSAPI_SECURITY_WPA_WPA2     = 0x4,      /*!< phrase conforms to WPA/WPA2 */
     NSAPI_SECURITY_PAP          = 0x5,      /*!< phrase conforms to PPP authentication context */
     NSAPI_SECURITY_CHAP         = 0x6,      /*!< phrase conforms to PPP authentication context */
+    NSAPI_SECURITY_EAP_TLS      = 0x7,      /*!< phrase conforms to EAP-TLS */
+    NSAPI_SECURITY_PEAP         = 0x8,      /*!< phrase conforms to PEAP */
```

Comment:
This is why it was functionality change, adding ne security types, and odin supports them?

Reply:
yes

Reply:
@0xc0170 enterprise mode was not previously supported it is added now so a functionality change